src/main/java/dk/cngroup/lentils/entity/view/CypherGameInfo.java

```diff
@@ -0,0 +1,49 @@
+package dk.cngroup.lentils.entity.view;
+
+import jdk.nashorn.internal.ir.annotations.Immutable;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Immutable
+@Table(name = "cypher_game_info")
```

Comment:
Toto nemuze vytvorit view ale vytvori to tabulku -> coz my nechceme.

Reply:
refactored na subselect